utils/init-google-map.ts

```
@@ -109,54 +113,146 @@ const updateStyles = (map: google.maps.Map, theme: 'light' | 'dark') => {
 	}
 };
 
-function createMapMarker (probe: Probe) {
+function createMapMarker (probe: Probe, showPulse: boolean = false, showIW: boolean = true, verticalOffset: number | null = null) {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)